docs/docs-components/Toc.js

```diff
@@ -69,6 +69,13 @@ function useThrottledOnScroll(callback: null | (() => void), delay: number) {
   }, [throttledCallback]);
 }
 
+type ToCItem = {|
+  id: string,
+  label: string,
+  // eslint-disable-next-line flowtype/no-mutable-array
```

Comment:
Can we fix this?

 $ReadOnlyArray<....>

Reply:
If we use readonly array then the algorithm [here](https://github.com/pinterest/gestalt/pull/3133/files#diff-8c97d01b6c6c7a51b95de107d690111f4e5485389b5166c8a2ff19135fc7fe4cR179) will not make sense. I tried making it readonly and I don't think coming up with "less straightforward" approach just to satisfy Flow is a good idea.

Reply:
give it a couple thoughts, we can merge and revisit this later on as a clean up